example/lib/main.dart

```diff
@@ -103,13 +116,15 @@ class _MyAppState extends State<MyApp> {
   @override
   void initState() {
     print("initState");
```

Comment:
this is now redundant, the log below is the same

Reply:
fixed